packages/@react-spectrum/link/docs/Link.mdx

```diff
@@ -0,0 +1,95 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/link';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Link} from '@react-spectrum/link';
+```
+
+# Link
+
+<p>{docs.exports.Link.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Link']}
+  sourceData={[{type: 'Spectrum', url: 'https://spectrum.adobe.com/page/link/'}]} />
+
+## Example
+
+```tsx example
+<Link onPress={() => console.log('pressed!')}>The missing link.</Link>
+```
+
+## Content
+
+A visible label is always required and can be provided by passing a child.
+
+```tsx example
+<Link>A label</Link>
+```
+
+To add an anchor (for example, to navigate to a location with an `href`), simply pass a anchor as a child.
+
+```tsx example
+<Link><a href="https://www.adobe.com">Adobe.com</a></Link>
+```
+
+The link component will clone its child and add the necessary styling and event handlers.
+
+### Internationalization
+
+In order to internationalize a link, a localized string should be passed to the `children` prop.
+
+### Accessibility
```

Comment:
this section needs a bit of help, especially since v2 docs don't really apply here (i'm pretty certain we dont turn links into anchors anymore, but instead clone the children).

Reply:
@majornista have opinions?